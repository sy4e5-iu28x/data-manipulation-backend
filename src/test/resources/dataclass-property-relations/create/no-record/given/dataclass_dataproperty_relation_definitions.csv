id, dataclass_id, dataproperty_id