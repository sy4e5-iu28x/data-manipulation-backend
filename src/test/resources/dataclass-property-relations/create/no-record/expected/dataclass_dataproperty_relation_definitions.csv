id, dataclass_id, dataproperty_id
1,2,2